content/docs/command-reference/add.md

```diff
@@ -153,10 +153,12 @@ not.
   > Additionally, this typically requires an external cache setup (see link
   > above).
 
-- `-o <path>`, `--out <path>` - destination `path` to make a local target copy,
-  or to [transfer](#example-transfer-to-cache) an external target into the cache
-  (and link to workspace). Note that this can be combined with `--to-remote` to
-  avoid storing the data locally, while still adding it to the project.
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target, which will now be tracked by DVC. Note that combining
+  this with an
+  [external cache transfer](#example-transfer-to-an-external-cache), or with the
+  `--to-remote` option, let's you avoid storing an external target locally,
+  while still adding it to the project.
 
 - `--to-remote` - import an external target, but don't move it into the
```

Comment:
`import` is not the right term here

Reply:
I would describe it as `when specified `dvc add` can accept path to an external data (e.g. S3 url, or path on a mounted volume, etc) and it copies data into remote storage w/o caching data into project's cache and w/o bringing it into the workspace. Besides that, it behaves in the same way - it creates a regular `.dvc` file, which can be used with `dvc pull` to get data into workspace when and where it's needed. This options expects (?) `-o` to be provided. It is useful to "bootstrap" the project, when data is too large for the disk the workspace is located on. See example ...`

Reply:
Good catch! Updated along those lines in d63b07f3.

> Besides that, it behaves in the same way...

This seems like too many details for an option desc. It's already explained in the main command Description and I think it's assumed that everything still applies with every option except if otherwise noted in each option desc.

> This options expects (?) -o

Not required, it should use `./<basename>` by default.

> useful to "bootstrap" ...

Similarly this concept is described in the example (linked from the option desc.)